### PR TITLE
Fixed disk migration issue

### PR DESCRIPTION
**Pull request by jianqiu**

### CODE REVIEW

File: softlayer/vm/softlayer_vm.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ const (
 	softLayerVMtag = "SoftLayerVM"
 	ROOT_USER_NAME = "root"
 	deleteVMLogTag = "DeleteVM"
+	ISCSIMANAGEMENT = "ISCSI Management"
```
Comment (maximilien) [inline softlayer/vm/softlayer_vm.go]: Change this to: `ISCSI_MANAGEMENT` since that would read better and follows convention

Comment (maximilien) [inline softlayer/vm/softlayer_vm.go]: Also, if these are all log tags then follow the convention like: `softLayerVMtag ` etc

File: softlayer/vm/softlayer_vm_test.go [hunk 2]
```diff
@@ -365,23 +365,35 @@ var _ = Describe("SoftLayerVM", func() {
 			disk bsldisk.Disk
 		)
 
-		const expectedTarget1 = `iqn.2001-05.com.equallogic:0-8a0906-ba580060c-fcc002c3743528a8-fake-user
+		const expectMultipathInstalled = `/sbin/multipath
```
Comment (maximilien) [inline softlayer/vm/softlayer_vm_test.go]: Contants on the other hand should be: `EXPECT_MULTIPATH_INSTALLED` etc

File: softlayer/vm/softlayer_vm.go [hunk 3]
```diff
@@ -370,11 +337,11 @@ func (vm SoftLayerVM) waitForVolumeAttached(virtualGuest datatypes.SoftLayer_Vir
 		return "", bosherr.WrapError(err, fmt.Sprintf("Failed to restart open iscsi from virtual guest `%d`", virtualGuest.Id))
 	}
 
+	if _, err = vm.discoveryOpenIscsiTargetsBasedOnShellScript(virtualGuest, volume); err != nil {
```
Comment (maximilien) [inline softlayer/vm/softlayer_vm.go]: Change this to: 

```
_, err = vm.discoveryOpenIscsiTargetsBasedOnShellScript(virtualGuest, volume)
if err != nil {
...
}
```

Comment (maximilien) [inline softlayer/vm/softlayer_vm.go]: Easier to read

**Comment by maximilien** [comment]
Please address comments and let me know and I will merge.

**Comment by jianqiu** [comment]
@maximilien  Thanks for your review and comments, I had adjusted our codes accordingly

**Comment by maximilien** [comment]
Cool, merging :beers:

**Pull request merged**